### PR TITLE
E2E Test Import

### PR DESCRIPTION
**Pull request by xmano**
Closes Issue#5589

### CODE REVIEW

**Comment by xmano** [comment]
Passes on CI:

File: apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+    cy.log("Select Contract type to upload");
+    cy.get('mat-form-field', {timeout: 30 * SEC})
+      .click();
+    
+    cy.get('mat-option')
+      .contains("Contract")
+      .click();
+      
+    //Import the Movie file here: filePicker
```
Comment (Mussuranae) [inline apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts:116]: Wrong comment ;)

File: apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    cy.log("Start upload by attaching the fixture");
+    cy.get('#filePicker', {timeout: 10 * SEC})
+      .attachFile(contractFixture);
+
+    cy.get('[test-id="status-import"]', { timeout: 30 * SEC })
+      .should('be.visible')
+      .should('contain', 'File uploaded');
+
+    cy.log("Movies uploaded successfully; Starting import..");
```
Comment (Mussuranae) [inline apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts:125]: Same, it's contracts that are imported, not movies here

**Review by Mussuranae — APPROVED** [review]

File: apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  cy.get('catalog-marketplace button[test-id=menu]', {timeout: 3 * SEC})
+    .first().click();
+  cy.get('aside a[routerlink="/c/o/dashboard/home"]', {timeout: 0.5 * SEC})
+    .click();
+  cy.wait(5 * SEC);
+  cy.get('aside a[routerlink="title"]', {timeout: 5 * SEC})
+    .click();
+
+  cy.get('catalog-dashboard h1', {timeout: 60 * SEC})
+    .contains("My Titles");
```
[the remark this answers is not in the document]
Comment (xmano) [reply apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts:38]: Yes we could do this. Will check

File: apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts (hunk 2 above)
Comment (xmano) [reply apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts:125]: Good catch. Will change.

File: apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts (hunk 1 above)
Comment (xmano) [reply apps/catalog/catalog-e2e/src/integration/dashboard/import.spec.ts:116]: fixed.

**Pull request merged**